runelite-client/src/main/java/net/runelite/client/plugins/blastfurnace/BlastFurnacePlugin.java

```diff
@@ -74,26 +93,63 @@ BlastFurnaceConfig provideConfig(ConfigManager configManager)
 	@Override
 	public Collection<Overlay> getOverlays()
 	{
-		return Arrays.asList(overlay, cofferOverlay, conveyorBeltOverlay);
+		return Arrays.asList(overlay, cofferOverlay, clickBoxOverlay);
+	}
+
+	@Subscribe
+	public void onItemContainerChanged(ItemContainerChanged event)
+	{
+		if (event.getItemContainer() != client.getItemContainer(InventoryID.EQUIPMENT))
+			return;
+
+		Item[] currentEquipment = event.getItemContainer().getItems();
+
+		for (Item item : currentEquipment)
+		{
+			if (item.getId() == ItemID.ICE_GLOVES)
+			{
+				iceGloves = true;
```

Comment:
At some point this would need to be set to false... I would maybe just check the equipment container from the overlay since that is fairly quick.